### PR TITLE
[MRG] Added Virtual Environment Directories to `.gitignore`

### LINKED ISSUE #738: Add Virtual Environment Directories to `.gitignore`
I'd like to propose adding the following lines to `.gitignore` file to prevent virtual environment directories from being tracked by Git:

```
# Virtual environments
env/
venv/
ENV/
```
## Explanation:
- Virtual environments are project-specific and contain dependencies that shouldn't be committed to the main repository.

# Benefits: 
- New contributors to this repo, **like myself**, won't have to worry about accidentally committing virtual environment directories, leading to a smoother onboarding experience.

### PR DESCRIPTION
**Pull request by samadpls**
Closes #738 

- Updated `.gitignore` to ignore Virtual Environment directories

### CODE REVIEW

**Review by gtdang — APPROVED** [review]
@rythorpe @jasmainak 
This looks fine to me provided there is a decision in the discussion of #738.

**Review by rythorpe — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,12 @@ coverage
 *.xml
 .venv
 
+# Virtual environments
```
Comment (jasmainak) [inline .gitignore:14]: I see a `.venv` above? That should also come here?

**Comment by jasmainak** [comment]
why does the linkcheck fail @samadpls ?

**Comment by samadpls** [comment]
> why does the linkcheck fail @samadpls ?

It seems like the linkcheck failure started after merging the master branch. I'll investigate further to determine if it's an issue specific to my changes or if other PRs are also encountering this problem.

**Comment by samadpls** [comment]
> `(auto_examples/workflows/plot_simulate_beta: line  330) broken    https://doi.org/10.1101/2021.04.16.440210 - 403 Client Error: Forbidden for url: https://www.biorxiv.org/lookup/doi/10.1101/2021.04.16.440210`
> 
> Linkcheck just fails if a URL is inaccessible correct? Seems like this one was down. Link seems to work for me now though. I can rerun the test. 

Yes… the linkcheck fails if a URL is inaccessible. Thank you for checking and rerunning the workflow. Looks  like everything's working fine now that it passed on the rerun.

**Comment by gtdang** [comment]
Thanks @samadpls!

**Pull request merged**